guide/src/web-getting-started.md

```diff
@@ -33,7 +33,7 @@ Then add `wasm()` and `topLevelAwait()` to the `plugins` section of your Vite co
 
 TODO
 
-## Add Ferrostar web components to your web app
+## Add Ferrostar web components to your React App
```

Comment:
Same as the above on being highly specific to react; the original headline is fine I think.

Reply:
Ok